### PR TITLE
New Port: Phantasy Star RL

### PR DESCRIPTION
**Pull request by Ganimoth**
# New Port for Phantasy Star RL

## Game Information
- **Title**: Phantasy Star RL
- **URL**: https://requinnzious.itch.io/phantasy-star-rl

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

File: ports/psrl/Phantasy Star RL.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# We log the execution of the script into log.txt
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+# Rename data.win
+[ -f "./data.win" ] && mv data.win game.droid
+
+$ESUDO chmod 666 /dev/uinput
+
+$GPTOKEYB "gmloadernext" -c "psrl.gptk"&
```
Comment (Slayer366) [inline ports/psrl/Phantasy Star RL.sh:35]:
```suggestion
$GPTOKEYB "gmloadernext" -c "psrl.gptk" &
```
I'm sure this was working, but I wonder if it'd be a good idea to put a space before the '&' so something goofy doesn't happen on another CFW/device...

**Review by Slayer366 — CHANGES_REQUESTED** [review]

**Review by Slayer366 — APPROVED** [review]

File: ports/psrl/Phantasy Star RL.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+cd $GAMEDIR
+
+# We log the execution of the script into log.txt
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+# Rename data.win
+[ -f "./data.win" ] && mv data.win game.droid
+
+$ESUDO chmod 666 /dev/uinput
```
Comment (Cebion) [inline ports/psrl/Phantasy Star RL.sh:33]:
```suggestion
```

File: ports/psrl/Phantasy Star RL.sh [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
```
Comment (Cebion) [inline ports/psrl/Phantasy Star RL.sh:34]:
```suggestion
pm_platform_helper "$GAMEDIR/gmloadernext"
```

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/psrl/Phantasy Star RL.sh [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+cd $GAMEDIR
+
+# We log the execution of the script into log.txt
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+# Rename data.win
+[ -f "./data.win" ] && mv data.win game.droid
+
+pm_platform_helper "$GAMEDIR/gmloadernext"
```
Comment (Cebion) [inline ports/psrl/Phantasy Star RL.sh:33]:
```suggestion
```

File: ports/psrl/Phantasy Star RL.sh [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+$GPTOKEYB "gmloadernext" -c "psrl.gptk" &
```
Comment (Cebion) [inline ports/psrl/Phantasy Star RL.sh:34]:
```suggestion
$GPTOKEYB "gmloadernext" -c "psrl.gptk" &
pm_platform_helper "$GAMEDIR/gmloadernext"
```

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/psrl/Phantasy Star RL.sh (hunk 5 above)
Comment (Cebion) [reply ports/psrl/Phantasy Star RL.sh:34]: Switched pm_platform_helper to after gptokeyb

**Review by Cebion — APPROVED** [review]

**Review by JanTrueno — APPROVED** [review]
👍

**Pull request merged**